src/WaterNetwork.jl

```diff
@@ -137,8 +141,11 @@ function constraintoffset_waternetwork_outflows(m::Model)
     end
 
     function generate(gg, tt)
-        # Determine number of gauges in county
-        b[gg, tt]
+        if get(config, "proportionnaturalflowforenvironment", nothing) == nothing
```

Comment:
Actually, I don't think this logic should be a part of WaterNetwork.  It should be an offset that defined in EnvironmentalFlows.